### PR TITLE
feat: ✨ Added order edit page design and functionality

### PR DESCRIPTION
**Pull request by ragulpalanisamy**
# Why?

- _Why are these changes required - if it is a new feature_
- _Issue - if the changes done is to fix some specific issue_

# Changes Done

- _Describe the changes being done_

# Testing

- _How did you test your changes ?_
- _Attach logs, screenshots (if applicable)_

# Jira Link

- _Add the link to the Story/Task for which this PR is being raised_

# References (if any)

- _Documentation Link_
- _Google Doc Links_
- _Other PRs_

### CODE REVIEW

File: client/src/components/Checkout/OrderEditPage.tsx [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+          <button
+            className="bg-secondary w-[40%] sm:w-[20%] lg:w-[10%] rounded-[5px] mr-2 p-3 whitespace-nowrap text-white"
+            onClick={() => handleOrderEdit()}
+          >
+            변경사항을
+          </button>
+          <button
+            className="bg-[#FF9494] p-3 w-[40%] sm:w-[20%] lg:w-[10%] rounded-[5px] text-white"
+            onClick={() => window.history.back()}
+          >
```
Comment (AbdulWadudh) [inline client/src/components/Checkout/OrderEditPage.tsx:258]: use `useNavigate()`
`const navigate = useNavigate();`
`onClick={() => navigate(-1)}`

File: client/src/components/Checkout/PaymentInformation.tsx [hunk 2]
```diff
@@ -1,11 +1,14 @@
 import * as Dialog from "@radix-ui/react-dialog";
+import { showErrorToast } from "../Toast/Toast";
 
 import Modal from "../Modal";
 import OrderReview from "./OrderReview";
 import Textarea from "../Form/Textarea";
+import { useNavigate } from "react-router-dom";
```
Comment (AbdulWadudh) [inline client/src/components/Checkout/PaymentInformation.tsx:8]: Rearrage the Import Statements

**Review by AbdulWadudh — CHANGES_REQUESTED** [review]

File: client/src/components/Checkout/OrderCard.tsx [hunk 3]
```diff
@@ -22,8 +22,8 @@ export default function OrderCard({
               type="radio"
               name="accountTransfer"
               className="my-auto"
-              checked={paymentMethod.includes("accountTransfer")}
-              onChange={() => setPaymentMethod("accountTransfer")}
+              checked={paymentMethod.includes("계좌이체")}
+              onChange={() => setPaymentMethod("계좌이체")}
```
[the remark this answers is not in the document]
Comment (ragulpalanisamy) [reply client/src/components/Checkout/OrderCard.tsx]: changes done

File: client/src/components/Checkout/OrderCard.tsx [hunk 4]
```diff
@@ -34,8 +34,8 @@ export default function OrderCard({
               type="radio"
               name="creditCard"
               className="my-auto"
-              onChange={() => setPaymentMethod("creditCard")}
-              checked={paymentMethod.includes("creditCard")}
+              onChange={() => setPaymentMethod("카드결제")}
+              checked={paymentMethod.includes("카드결제")}
```
[the remark this answers is not in the document]
Comment (ragulpalanisamy) [reply client/src/components/Checkout/OrderCard.tsx]: changes done

File: client/src/components/Checkout/OrderDetails.tsx [hunk 5]
```diff
@@ -93,6 +93,15 @@ export default function OrderDetails() {
     thumbnail: orderDetails?.data?.order_item[0]?.thumbnail,
     productName: orderDetails?.data?.order_item[0]?.product_name,
     productId: orderDetails?.data?.order_item[0]?.product_id,
+    deliveryTracking: orderDetails?.data?.shipment[0]?.tracking_number,
+    paymentMethod: orderDetails?.data?.payment_method,
+    orderId: orderId,
```
[the remark this answers is not in the document]
Comment (ragulpalanisamy) [reply client/src/components/Checkout/OrderDetails.tsx:98]: changes done

File: client/src/components/Checkout/OrderEditPage.tsx [hunk 6]
```diff
@@ -0,0 +1,250 @@
...
+        method: "post",
+        data: orderEdit,
+      });
+      if (res.status) {
+        setOrderEdit(res.data);
+        showSuccessToast("주문 편집가능합니다.");
+      }
+    } catch (error) {
+      console.error("Error fetching data:", error);
+    }
```
[the remark this answers is not in the document]
Comment (ragulpalanisamy) [reply client/src/components/Checkout/OrderEditPage.tsx]: changes done

File: client/src/components/Checkout/OrderEditPage.tsx (hunk 1 above)
Comment (ragulpalanisamy) [reply client/src/components/Checkout/OrderEditPage.tsx:258]: changes done

File: client/src/components/Checkout/PaymentInformation.tsx (hunk 2 above)
Comment (ragulpalanisamy) [reply client/src/components/Checkout/PaymentInformation.tsx:8]: changes done

**Review by AbdulWadudh — APPROVED** [review]

**Pull request merged**